_maps/map_files/BirdStation/job_changes.dm

```diff
@@ -0,0 +1,138 @@
+#define JOB_MODIFICATION_MAP_NAME "BirdStation"
+
+//Removed jobs
+MAP_REMOVE_JOB(atmos)
+MAP_REMOVE_JOB(chief_engineer)
+MAP_REMOVE_JOB(qm)
+MAP_REMOVE_JOB(cmo)
+MAP_REMOVE_JOB(chemist)
+MAP_REMOVE_JOB(geneticist)
+MAP_REMOVE_JOB(virologist)
+MAP_REMOVE_JOB(rd)
+MAP_REMOVE_JOB(roboticist)
+MAP_REMOVE_JOB(chaplain)
+MAP_REMOVE_JOB(warden)
+MAP_REMOVE_JOB(lawyer)
+
+//Job changes
+
+/datum/job/New()
+	..()
+	MAP_JOB_CHECK
+	supervisors = "the captain and the head of personnel"
+
+
+/datum/job/assistant // Here so assistant appears on the top of the select job list.
+
+//Access Changes + Flavo(u)r
+//Command
+
+/datum/job/captain/New()
+	..()
+	MAP_JOB_CHECK
+	supervisors = "Nanotrasen and Central Command"
+
+/datum/job/hop/New()
+	..()
+	MAP_JOB_CHECK
+	supervisors = "the captain and Central Command"
+
+/datum/job/hop/get_access()
+	MAP_JOB_CHECK
```

Comment:
This is causing HOP to have no access on all maps except BirdStation, look at the OmegaStation removal for what it should be